app/helpers/application_helper.rb

```diff
@@ -4,6 +4,18 @@ def company_dropdown
     Company.all.map { |company| [company.name, company.id] }.sort
   end
 
+  def company_logo
+    current_employee.company.logo if current_employee
+  end
+
+  def primary_color
+    current_employee.company.primary_color if current_employee
+  end
+
+  def secondary_color
+    current_employee.company.secondary_color if current_employee
+  end
+
```

Comment:
I abstracted out the code from the view into a easier to use helper that first checks if the current_employee exists before trying to call `.company` on it. Saving a lot of errors and breaking tests.